addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17573,10 +17588,10 @@ msgctxt "#35258"
 msgid "Select emulator for {0:s} file"
 msgstr ""
 
-#. Label on emulators with a savestate in the emulator selection dialog
-#: xbmc/games/dialogs/GUIDialogSelectGameClient.cpp
+#. Title of the dialog to select a savestate for the current game
+#: xbmc/games/dialogs/GUIDialogSelectSavestate.cpp
 msgctxt "#35259"
-msgid "Saved"
+msgid "Select savestate"
```

Comment:
Can you make this a new ID, and change the one for `Saved` to empty? Transifix does better with adding new strings than changing existing ones.